scenarios/tracking/FAQ.md

```diff
@@ -0,0 +1,173 @@
+# Multi-Object Tracking
+
+```diff
++ June 2020: This work is ongoing.
+```
+
+## Frequently asked questions
+
+This document tries to answer frequent questions related to multi-object tracking. For generic Machine Learning questions, such as "How many training examples do I need?" or "How to monitor GPU usage during training?" see also the image classification [FAQ](https://github.com/microsoft/ComputerVision/blob/master/classification/FAQ.md).
+
+* General
+  * [Why FairMOT repository for the tracking algorithm?](#why-FAIRMOT)
+  * [What are additional complexities that can enhance the current MOT algorithm](#What-are-additional-complexities-that-can-enhance-the-current-MOT-algorithm)
+  * [What is the difference between online and offline (batch) tracking algorithms?](#What-is-the-difference-between-online-and-offline-tracking-algorithms)
+
+* Data  
+  * [How to annotate a video for evaluation?](#how-to-annotate-a-video-for-evaluation)
+  * [What is the MOT Challenge format used by the evaluation package?](#What-is-the-MOT-Challenge-format-used-by-the-evaluation-package)
+
+* Technology State-of-the-Art (SoTA)
+  * [What is the architecture of the FairMOT tracking algorithm?](#What-is-the-architecture-of-the-FairMOT-tracking-algorithm)
+  * [What are SoTA object detectors used in tracking-by-detection trackers?](#What-are-SoTA-object-detectors-used-in-tracking-by-detection-trackers) 
+  * [What are SoTA feature extraction techniques used in tracking-by-detection trackers?](#What-are-SoTA-feature-extraction-techniques-used-in-tracking-by-detection-trackers)
+  * [What are SoTA affinity and association techniques used in tracking-by-detection trackers?](#What-are-SoTA-affinity-and-association-techniques-used-in-tracking-by-detection-trackers)
+  * [What are the main evaluation metrics for tracking performance?](#What-are-the-main-evaluation-metrics)
+
+* Training and Inference
+  * [How to improve training accuracy?](#how-to-improve-training-accuracy)
+  * [What are the main training parameters in FairMOT](#what-are-the-main-training-parameters-in-FairMOT)
+  * [What are the main inference parameters in FairMOT?](#What-are-the-main-inference-parameters-in-FairMOT])
+  * [What are the training losses for MOT using FairMOT?](#What-are-the-training-losses-for-MOT-using-FairMOT? )
+
+* MOT Challenge
+  * [What is the MOT Challenge?](#What-is-the-MOT-Challenge)
+
+
+
+
+## General
+
+### Why FairMOT?
+FairMOT is an [open-source](https://github.com/ifzhang/FairMOT) online one-shot tracking algorithm, that has shown [competitive performance in recent MOT benchmarking challenges](https://motchallenge.net/method/MOT=3015&chl=5), at fast inference speed.  
```

Comment:
"one-shot tracking algorithm that has shown"  no comma and no comma 

also i think is should be 'inferencing speeds' ?